### ISSUE TITLE
Clean up commented line

### ISSUE THREAD

**Issue by kevinrue** [iSEE/iSEE#646]
https://github.com/iSEE/iSEE/pull/645/files/7384c0ae4497ae4968cd6a35ad4a50de29e54249#diff-8e38bb8dcb100c78cbedf8aa25774eec3bcb718344fd8b856005f4d7dd912fe3R130

@federicomarini 

post-release is good too

**Comment by federicomarini** [comment]
Good point - thing is, maybe we want to uncomment it, if we ever think that the json export is something to have.
But let's get the stormy days of pre-release pass by 😉

**Comment by kevinrue** [comment]
In that case, please leave a comment right in the lines right above those lines to say so. Otherwise, the next person who comes across those lines will wonder again why they're there but commented out.
